src/client/components/MyNotification/MyNotification.tsx

```diff
@@ -19,6 +20,12 @@ type MyNotificationProps = {
   smallVariant?: true;
 };
 
+function someWhatReadableThemaTitleFromID(themaID?: string) {
+  return capitalizeFirstLetter(
+    (themaID ?? 'Mijn Amsterdam').toLowerCase().replace(/_/g, ' ')
```

Comment:
Haha, ik hou van de functie naam.

[nit]
```suggestion
    (themaID ?? 'Mijn Amsterdam').toLowerCase().replaceAll('_', ' ')
```